### PR TITLE
Ingame screen constant mappings

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

**Review by liach — APPROVED** [review]
Sure...

File: mappings/net/minecraft/client/gui/screen/ingame/StonecutterScreen.mapping [hunk 1]
```diff
@@ -4,6 +4,15 @@ CLASS net/minecraft/class_3979 net/minecraft/client/gui/screen/ingame/Stonecutte
...
 	FIELD field_17674 scrollAmount F
+	FIELD field_32372 SCROLLBAR_WIDTH I
+	FIELD field_32373 SCROLLBAR_HEIGHT I
+	FIELD field_32374 RECIPE_LIST_COLUMNS I
+	FIELD field_32375 RECIPE_LIST_ROWS I
+	FIELD field_32376 RECIPE_ENTRY_WIDTH I
+	FIELD field_32377 RECIPE_ENTRY_HEIGHT I
+	FIELD field_32378 SCROLLBAR_AREA_HEIGHT I
+	FIELD field_32379 RECIPE_LIST_X_OFFSET I
+	FIELD field_32380 RECIPE_LIST_Y_OFFSET I
```
Comment (Juuxel) [inline mappings/net/minecraft/client/gui/screen/ingame/StonecutterScreen.mapping]:
```suggestion
	FIELD field_32379 RECIPE_LIST_OFFSET_X I
	FIELD field_32380 RECIPE_LIST_OFFSET_Y I
```
Yarn conventions: "Coordinates can be named `x`, `y`, and `z` when it's clear what they represent. If clarification is needed, add a word in front of the coordinate (`velocityX`, not `xVelocity`)."

File: mappings/net/minecraft/client/gui/screen/ingame/MerchantScreen.mapping [hunk 2]
```diff
@@ -7,6 +7,21 @@ CLASS net/minecraft/class_492 net/minecraft/client/gui/screen/ingame/MerchantScr
...
+	FIELD field_32360 SECOND_BUY_ITEM_X_OFFSET I
+	FIELD field_32361 SOLD_ITEM_X_OFFSET I
+	FIELD field_32363 MAX_TRADE_OFFERS I
+	FIELD field_32365 TRADE_OFFER_BUTTON_HEIGHT I
+	FIELD field_32366 TRADE_OFFER_BUTTON_WIDTH I
+	FIELD field_32367 SCROLLBAR_HEIGHT I
+	FIELD field_32368 SCROLLBAR_WIDTH I
+	FIELD field_32369 SCROLLBAR_AREA_HEIGHT I
+	FIELD field_32370 SCROLLBAR_Y_OFFSET I
+	FIELD field_32371 SCROLLBAR_X_OFFSET I
```
Comment (Juuxel) [inline mappings/net/minecraft/client/gui/screen/ingame/MerchantScreen.mapping]:
```suggestion
	FIELD field_32370 SCROLLBAR_OFFSET_Y I
	FIELD field_32371 SCROLLBAR_OFFSET_X I
```

File: mappings/net/minecraft/client/gui/screen/ingame/MerchantScreen.mapping [hunk 3]
```diff
@@ -7,6 +7,21 @@ CLASS net/minecraft/class_492 net/minecraft/client/gui/screen/ingame/MerchantScr
 	FIELD field_26570 SEPARATOR_TEXT Lnet/minecraft/class_2561;
 	FIELD field_26571 DEPRECATED_TEXT Lnet/minecraft/class_2561;
 	FIELD field_2950 TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_32354 TEXTURE_WIDTH I
+	FIELD field_32355 TEXTURE_HEIGHT I
+	FIELD field_32357 XP_BAR_X_OFFSET I
+	FIELD field_32358 TRADE_LIST_AREA_Y_OFFSET I
+	FIELD field_32359 FIRST_BUY_ITEM_X_OFFSET I
+	FIELD field_32360 SECOND_BUY_ITEM_X_OFFSET I
+	FIELD field_32361 SOLD_ITEM_X_OFFSET I
```
Comment (Juuxel) [inline mappings/net/minecraft/client/gui/screen/ingame/MerchantScreen.mapping:16]:
```suggestion
	FIELD field_32357 XP_BAR_OFFSET_X I
	FIELD field_32358 TRADE_LIST_AREA_OFFSET_Y I
	FIELD field_32359 FIRST_BUY_ITEM_OFFSET_X I
	FIELD field_32360 SECOND_BUY_ITEM_OFFSET_X I
	FIELD field_32361 SOLD_ITEM_OFFSET_X I
```
Edit: nevermind, see liach's comment below

File: mappings/net/minecraft/client/gui/screen/ingame/LoomScreen.mapping [hunk 4]
```diff
@@ -12,6 +12,14 @@ CLASS net/minecraft/class_494 net/minecraft/client/gui/screen/ingame/LoomScreen
...
 	FIELD field_2968 scrollPosition F
 	FIELD field_2970 firstPatternButtonId I
+	FIELD field_32346 PATTERN_LIST_COLUMNS I
+	FIELD field_32347 PATTERN_LIST_ROWS I
+	FIELD field_32348 SCROLLBAR_WIDTH I
+	FIELD field_32349 SCROLLBAR_HEIGHT I
+	FIELD field_32350 PATTERN_ENTRY_SIZE I
+	FIELD field_32351 SCROLLBAR_AREA_HEIGHT I
+	FIELD field_32352 PATTERN_LIST_X_OFFSET I
+	FIELD field_32353 PATTERN_LIST_Y_OFFSET I
```
Comment (Juuxel) [inline mappings/net/minecraft/client/gui/screen/ingame/LoomScreen.mapping]:
```suggestion
	FIELD field_32352 PATTERN_LIST_OFFSET_X I
	FIELD field_32353 PATTERN_LIST_OFFSET_Y I
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/gui/screen/ingame/MerchantScreen.mapping (hunk 3 above)
Comment (liach) [reply mappings/net/minecraft/client/gui/screen/ingame/MerchantScreen.mapping:16]: Those offsets don't have a corresponding x/y component, so I think they are more like "x/y offset" than "x/y of the offset"

**Comment by IotaBread** [comment]
Oh no, I deleted the wrong branch

**Review by enbrain — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**